tracer/src/Datadog.Trace/Debugger/Helpers/DisposableEnumerable.cs

```diff
@@ -0,0 +1,33 @@
+// <copyright file="DisposableEnumerable.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace Datadog.Trace.Debugger.Helpers
+{
+    internal struct DisposableEnumerable<T> : IDisposable
```

Comment:
I think we can (should) make this a `readonly struct`
```suggestion
    internal readonly struct DisposableEnumerable<T> : IDisposable
```